tests/test_config.py

```diff
@@ -15,34 +26,36 @@
 
 
 @pytest.fixture
-def mocked_logging_config_module(mocker):
+def mocked_logging_config_module(mocker: MockerFixture) -> MagicMock:
     return mocker.patch("logging.config")
 
 
 @pytest.fixture(scope="function")
-def logging_config():
+def logging_config() -> dict:
     return deepcopy(LOGGING_CONFIG)
 
 
 @pytest.fixture
-def json_logging_config(logging_config):
+def json_logging_config(logging_config: dict) -> str:
     return json.dumps(logging_config)
 
 
 @pytest.fixture
-def yaml_logging_config(logging_config):
+def yaml_logging_config(logging_config: dict) -> str:
     return yaml.dump(logging_config)
 
 
-async def asgi_app(scope, receive, send):
+async def asgi_app(
+    scope: Scope, receive: ASGIReceiveCallable, send: ASGISendCallable
+) -> None:
     pass  # pragma: nocover
 
 
-def wsgi_app(environ, start_response):
+def wsgi_app(environ: Any, start_response: Any) -> None:
```

Comment:
I've made a draft PR that touched a little bit on wsgi app and came up with
```
Environ = typing.MutableMapping[str, typing.Any]
ExcInfo = typing.Tuple[    typing.Type[BaseException], BaseException, typing.Optional[TracebackType]]
StartResponse = typing.Callable[[str, typing.Iterable[typing.Tuple[str, str]], typing.Optional[ExcInfo]], None]
```

```suggestion
def wsgi_app(environ: Environ, start_response: StartResponse) -> None:
```

Reply:
These are good suggestions. For the `Environ` type, I would modify it slightly, either simply `typing.MutableMapping` (which is what Starlette uses in [starlette/config.py](https://github.com/encode/starlette/blob/b6f3578bb2cf6c60e3efe110143409b47f368d36/starlette/config.py#L16)), or `typing.MutableMapping[str, str]`, based on [`os.environ`](https://github.com/python/cpython/blob/3e1c7167d86a2a928cdcb659094aa10bb5550c4c/Lib/os.py#L737) (which appears to expect all keys and values to be strings).

I went with `typing.MutableMapping`, hopefully that's cool.

Reply:
the starlette Environ you're referring to has nothing to do with the Environ here I think, appart from the name, the latter is for configuring starlette iirc, the former is the dict for the wsgi spec (https://www.python.org/dev/peps/pep-0333/#environ-variables)
I think `typing.MutableMapping[str, str]` is incorrect since one of the values it can takes can be bytes, for the wsgi.input key

Reply:
Ok that's fine, I accepted your suggestion (6e1c1dc).